src/sage/modules/vector_modn_dense.pyx

```diff
@@ -168,19 +168,32 @@ cdef class Vector_modn_dense(free_module_element.FreeModuleElement):
             self._init(parent.degree(), parent, parent.base_ring().order())
 
     def __init__(self, parent, x, coerce=True, copy=True):
+        """
+        Create an element.
+
+        Note that ``coerce=False`` is dangerous::
+
+            sage: V = VectorSpace(GF(7), 3)
+            sage: v = V([2, 9, -5], coerce=False)
+            sage: v[0] == v[1]
+            False
+            sage: v[0]+1 == v[1]+1
+            True
+            sage: v[0] == v[2]
+            False
+        """
         cdef Py_ssize_t i
-        cdef mod_int a, p
+        cdef mod_int a
         if isinstance(x, xrange):
```

Comment:
please replace xrange by range here

Reply:
Okay, but apparently it's intentional that Cython supports `xrange` (will just be replaced by `range`)

https://github.com/cython/cython/issues/1036